app-vim/lightline/lightline-0_pre20221005.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit vim-plugin
+
+LIGHTLINE_COMMIT=b1e91b41f5028d65fa3d31a425ff21591d5d957f
```

Comment:
I prefer that this constant is moved into the `else` block, closer to usage, less places to miss/break.

Reply:
done, please check